src/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj

```diff
@@ -13,8 +13,7 @@
     <IsPackable>true</IsPackable>
     <VersionPrefix>4.5.0</VersionPrefix>
     <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.6.0</VersionPrefix>
-    <AssemblyVersion Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.1.4.0</AssemblyVersion>
-    <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.1.5.0</AssemblyVersion>
+    <AssemblyVersion Condition="'$(IsPackable)' == 'true'">4.1.5.0</AssemblyVersion>
```

Comment:
For this one we should keep it pinned, because `netcoreapp2.1` where the package expects the framework to provide the implementation, is 
```
// C:\Users\ericstj\.nuget\packages\microsoft.netcore.app\2.0.0\ref\netcoreapp2.0\System.Numerics.Vectors.dll
// System.Numerics.Vectors, Version=4.1.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
```

I see we're pinning to `4.1.4.0` though, which looks like it's already higher than the one that shipped inbox in 2.0.  It seems that previously the reference assembly was pinned to `4.1.3.0`
```
// C:\Users\ericstj\.nuget\packages\system.numerics.vectors\4.5.0\ref\netstandard2.0\System.Numerics.Vectors.dll
// System.Numerics.Vectors, Version=4.1.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
```